src/client/components/EmailInput.tsx

```diff
@@ -67,10 +68,17 @@ export const EmailInput: React.FC<EmailInputProps> = ({
       name="email"
       type="email"
       error={errorMessage}
+      cssOverrides={disableAutofillBackground}
       onBlur={(e) => {
         // Transition error state when the input box loses focus.
         transitionState(e.target.validity);
       }}
+      onKeyDown={(e) => {
+        if (e.code === '') {
+          e.currentTarget.checkValidity();
+          transitionState(e.currentTarget.validity);
+        }
+      }}
```

Comment:
Is this unrelated to the lastpass fixes?

Reply:
this one is so that when LastPass autofills the email field we run a validation check